Replicator/RevFinder.cc

```diff
@@ -141,7 +142,7 @@ namespace litecore::repl {
 
                 auto &encoder = response.jsonBody();
                 encoder.beginArray();
-                int requested = proposed ? findProposedRevs(changes, encoder, sequences)
+                int requested = proposed ? findProposedRevs(changes, encoder, req->boolProperty(Pusher::kConflictIncludesRevProperty), sequences)
```

Comment:
This line looks way longer than 100 chars

Reply:
Oh!  I wasn't yet aware of this style guideline, I'll keep that in mind.